docs/open-source-tizen/tools/gbs/gbs-faq.md

```diff
@@ -1,94 +1,110 @@
-# GBS Frequently Asked Questions
+# FAQ
```

Comment:
I decided the tile after investigating many web sites.

Please keep the base commit.

Reply:
OK. We will change the MIC FAQ topic title to "MIC Frequently Asked Questions" too.